server/views/partials/layout.njk

```diff
@@ -100,6 +100,11 @@
   <script src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
   <script src="/assets/print.js"></script>
+  <script src="/assets/applicationinsights-web.min.js"></script>
+  <script src="/assets/applicationinsights.js"></script>
```

Comment:
This line loads our custom JS file which makes the object `applicationInsights` available